scripts/variantstore/wdl/GvsCreateVATfromVDS.wdl

```diff
@@ -527,11 +527,16 @@ task AnnotateVCF {
         # Mentioning this path in the inputs section of the task combined with checking the 'Use reference disks' option
         # in Terra UI tells Cromwell to arrange for the Nirvana reference disk to be attached to this VM.
         File summon_reference_disk =
-            "gs://broad-public-datasets/gvs/vat-annotations/Nirvana/3.18.1/SupplementaryAnnotation/GRCh38/MITOMAP_20200819.nsa.idx"
+            "gs://gcp-public-data--broad-references/hg38/v0/Nirvana/3.18.1_2024-03-06/SupplementaryAnnotation/GRCh38/MITOMAP_20200819.nsa.idx"
```

Comment:
Do you have a link to the config PR you created so I can check this matches the new reference disk config?

Reply:
Here's the PR in terra-helmfile: https://github.com/broadinstitute/terra-helmfile/pull/5254